Sprout/View Controllers/ComposePlanViewController.m

```diff
@@ -6,26 +6,74 @@
 //
 
 #import "ComposePlanViewController.h"
+#import "Goal.h"
+#import "Networker.h"
+#import "Task.h"
 
 @interface ComposePlanViewController ()
 
+@property (weak, nonatomic) IBOutlet UISegmentedControl *timeframeControl;
+@property (weak, nonatomic) IBOutlet UIButton *saveTimeframe;
+@property (weak, nonatomic) IBOutlet UITextField *taskNameField;
+@property (weak, nonatomic) IBOutlet UISegmentedControl *taskTypeControl;
+@property (weak, nonatomic) IBOutlet UISegmentedControl *taskFrequencyControl;
+@property (weak, nonatomic) IBOutlet UIButton *addTaskButton;
+@property (weak, nonatomic) IBOutlet UIButton *saveGoal;
+@property (nonatomic, strong) NSNumber *timeframe;
+@property (nonatomic, strong) NSMutableArray<Task *> *arrayOfTasks;
+
 @end
 
 @implementation ComposePlanViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    // Do any additional setup after loading the view.
+    
+}
+- (IBAction)saveTimeFrame:(id)sender {
```

Comment:
I would use 'set' instead of 'save'. Save implies to the reader that it will be put into some kind of persistent storage.